### PR TITLE
fix(protocol-designer): misc bugs with copy, ui, and liquid color error

### PR DESCRIPTION
**Pull request by jerader**
closes AUTH-386 AUTH-387 AUTH-385 partially RQA-2682

# Overview

This PR addresses a handful of random things found in candidate-A for PD 8.1:
1. fixes ui for multiple tipracks to match the modules page
2. update tuberack warning
3. fix whitescreen when entering an invalid color hex code
4. update an image to a transparent img

# Test Plan

- Create an ot-2 protocol and choose a pipette with more than 3 tiprack options. select all 3 tiprack options and see that the rest of the options are disabled and the checkbox disappears. Then on the module page, see that the thermocycler gen1 does not have a white background (i only partially addressed the module images since i'm waiting for design to provide the remaining module images with a transparent background). 
- Then go to liquids tab and try to add an invalid color of `000000`, see that there is an error message saying its an invalid color. 
- Finally, go to the deck map tab and add a tuberack. Add a transfer step and specify the aspirate and dispense to occur in the tuberack. See that the warning banner text says that the default aspirate and dispense height is `1mm`.

# Changelog

- update copy and an image
- fix bug with rendering the error text for liquid colors
- expand `equipmentOption` to include the pipette tip type and adjust the tooltip text and disabled state

# Review requests

see test plan

# Risk assessment

low

### CODE REVIEW

File: protocol-designer/src/components/LiquidsPage/LiquidEditForm.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export function LiquidEditForm(props: LiquidEditFormProps): JSX.Element {
       serialize: values.serialize || false,
     })
   }
-
+  console.log('errors', errors)
```
Comment (koji) [inline protocol-designer/src/components/LiquidsPage/LiquidEditForm.tsx]: need to keep for debugging?

**Review by koji — APPROVED** [review]
The changes look good to me. The sandbox worked as expected.
Once the images are replaced, this pr is good to go.

File: protocol-designer/src/components/LiquidsPage/LiquidEditForm.tsx (hunk 1 above)
Comment (jerader) [reply protocol-designer/src/components/LiquidsPage/LiquidEditForm.tsx]: ooops forgot to delete it, thank you!

**Comment by jerader** [comment]
> The changes look good to me. The sandbox worked as expected. Once the images are replaced, this pr is good to go.

@koji thanks! i think i'll fix the images in a follow up since i'm still waiting for design to give me the new images with the transparent background. In that followup i'll also update all the images to the optimized versions you made.

**Comment by koji** [comment]
> > The changes look good to me. The sandbox worked as expected. Once the images are replaced, this pr is good to go.
> 
> @koji thanks! i think i'll fix the images in a follow up since i'm still waiting for design to give me the new images with the transparent background. In that followup i'll also update all the images to the optimized versions you made.

that totally makes sense

**Pull request merged**